opentelekomcloud/services/cbr/resource_opentelekomcloud_cbr_vault_v3.go

```diff
@@ -216,12 +216,7 @@ func ResourceCBRVaultV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
-			"tags": {
-				Type:         schema.TypeMap,
-				Optional:     true,
-				ForceNew:     true,
-				ValidateFunc: common.ValidateTags,
```

Comment:
revert validation for this time, it's a bad idea to left it to the server

Reply:
common.TagsSchema() contain the ValidateFunc